apps/condo/domains/ticket/hooks/useTicketTableFilters.tsx

```diff
@@ -566,5 +566,5 @@ export function useTicketTableFilters (): Array<FiltersMeta<TicketWhereInput>>
                 filters: [filterTicketContact],
             },
         ]
-    }, [statuses, sources])
+    }, [statuses, sources, categoryClassifiers])
```

Comment:
It's also would be good to add all dependencies here as it already done in callcenter.
<img width="1916" alt="image" src="https://user-images.githubusercontent.com/25384290/184827443-8618d6c6-e776-4b9c-bcb3-f272e241a447.png">